ParisBNF/abb/BNFabb166.xml

```diff
@@ -0,0 +1,500 @@
+                                    </incipit>
+                            </msItem>
+                         
+                        </msContents>
+                        
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf" quantity="2">2</measure>
+                                        <locus target="#i"/>
+                                        <locus target="#1"/>
+                                    </extent>
+                                </supportDesc>
+                                    
+                                                            
+                        <layoutDesc>
+                            <layout columns="3" writtenLines="27 28">
+                                <locus from="1ra" to="1vc"/>                                                                    
+                                <ab type="pricking">Pricking is visible.</ab>
+                                <ab type="ruling">Ruling is visible.</ab>
+                                <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                                <note>The leaf<locus target="#i"/> has been neither pricked no ruled.</note></layout>
+                        </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic" corresp="#p1 #p3">
+                                <locus from="1ra" to="1vc"/>                                
+                                <seg type="script">18th century</seg>                            
+                                <seg type="ink">Black; red.</seg>                     
+                                <desc>Careful, very regular; by a well-trained scribe.</desc>
+                            </handNote>
+                        </handDesc>
+                            <additions>
+                                <list>
+                                    
+                                    <item xml:id="a1">
+                                        <locus from="iv"/>
+                                        <desc type="Unclear">An unclear note</desc>
+                                        <q xml:lang="gez"/>
+                                    </item>
+                                    
+                                    <item xml:id="e1">
+                                        <locus from="1r"/>
+                                        <desc type="StampExlibris">Shelfmark and stamp of the Library</desc>
```

Comment:
I think this addition is recent and belongs to the entire MS and not to this msPart, move up to the first physDesc

Reply:
The other way around: when we know for sure that an addition was added before the parts were merged, then we encode it with the part, as a part is a distinct layer in production.
When we can assume that they were added later OR whenever we are not sure we put them with the main physDesc, not with the part, as the main description is for the entire MS.